docs/index.md

```diff
@@ -1,31 +1,23 @@
 Zanata Server
 =============
+Please visit [our about page](http://zanata.org/about/)
 
-TBD
-
-Features
---------
-
-- TBD
 
 Installation
 ------------
 
-<!-- For installation instructions please see the [Installation Section](installation) -->
+[Installation Section](installation)
 
 Contribute
 ----------
 
-- Issue Tracker: http://bugzilla.redhat.com
-- Source Code: github.com/zanata/zanata-server
+- Issue Tracker: [Bugzilla](http://bugzilla.redhat.com/buglist.cgi?product=Zanata)
+- Source Code: [GitHub](http://github.com/zanata)
 
 Support
 -------
-
-If you are having issues, please let us know.
-We have a mailing list located at: zanata-users@redhat.com
+Click [here](http://zanata.org/help/) for help and support
 
 License
 -------
-
-<!-- The project is licensed under the BSD license. -->
\ No newline at end of file
+Click [here](https://github.com/zanata/zanata-server/blob/master/README.md) for our license information.
```

Comment:
That is the readme, not the license file. We should at least copy the section from the readme so people don't have to click and search for the info:

```
Zanata is Free software, licensed under the [LGPL](http://www.gnu.org/licenses/lgpl-2.1.html).
```